x/meshsecurity/client/cli/gov_tx.go

```diff
@@ -0,0 +1,136 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	errorsmod "cosmossdk.io/errors"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	v1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/osmosis-labs/mesh-security-sdk/x/meshsecurity/types"
+)
+
+// DefaultGovAuthority is set to the gov module address.
+// Extension point for chains to overwrite the default
+var DefaultGovAuthority = sdk.AccAddress(address.Module("gov"))
+
+func SubmitProposalCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "submit-proposal",
+		Short:        "Submit a mesh security proposal.",
+		SilenceUsage: true,
+	}
+	cmd.AddCommand(
+		ProposalSetVirtualStakingMaxCapCmd(),
+	)
+	return cmd
+}
+
+func ProposalSetVirtualStakingMaxCapCmd() *cobra.Command {
+	bech32Prefix := sdk.GetConfig().GetBech32AccountAddrPrefix()
+	cmd := &cobra.Command{
+		Use:   "set-virtual-staking-max-cap [contract_addr_bech32] [max_cap] --title [text] --summary [text] --authority [address]",
+		Short: "Submit a set virtual staking max cap proposal",
+		Args:  cobra.ExactArgs(2),
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Submit a proposal to set virtual staking maximum cap limit to the given contract.
+
+Example:
+$ %s tx meshsecurity submit-proposal set-virtual-staking-max-cap %s1l94ptufswr6v7qntax4m7nvn3jgf6k4gn2rknq 100stake --title "a title" --summary "a summary" --authority %s1eppn90g4eg7guvyhz0ayph40c3e6uktygdd2nq
```

Comment:
pp: would it make sense to use the DefaultGovAuthority address for the authority example?